compilers/concrete-compiler/compiler/lib/Support/CompilerEngine.cpp

```diff
@@ -914,9 +914,10 @@ llvm::Expected<std::string> CompilerEngine::Library::emitShared() {
       extraArgs.push_back("-rpath " + rpath);
     }
 #else // Linux
-    extraArgs.push_back(runtimeLibraryPath);
-    if (!rpath.empty()) {
-      extraArgs.push_back("-rpath=" + rpath);
+    if (!rpath.empty() && !runtimeLibraryName.empty()) {
```

Comment:
what happens if we have only runtimeLibraryName ? (as before) ?

Reply:
I'm not very certain about details at this point anymore but I remember adding it when testing if the image filter example application would work.